introduction.md

```diff
@@ -0,0 +1 @@
+Hey, I'm Curtis, new to the Dojo and fairly new to Git!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.